src/mammos_ai/_beyond_stoner_wohlfarth_fixed_angle/__init__.py

```diff
@@ -0,0 +1,232 @@
+"""Functions to predict properties related to hysteresis loops."""
+
+from __future__ import annotations
+
+from pathlib import Path
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    import numbers
+
+    import astropy.units
+    import mammos_entity
+
+import mammos_analysis
+import mammos_entity as me
+import mammos_units as u
+import numpy as np
+import onnxruntime as ort
+
+MODEL_DIR = Path(__file__).parent
+
+MODELS = {
+    "classifier": MODEL_DIR / "classifier_cube50_singlegrain_random_forest_v0.1.onnx",
+    False: MODEL_DIR / "soft_cube50_singlegrain_random_forest_v0.1.onnx",
+    True: MODEL_DIR / "hard_cube50_singlegrain_random_forest_v0.1.onnx",
+}
+
+_SESSION_OPTIONS = ort.SessionOptions()
+_SESSION_OPTIONS.log_severity_level = 3
+
+
+def is_hard_magnet_from_Ms_A_K(
+    Ms: mammos_entity.Entity | astropy.units.Quantity | numbers.Number | np.ndarray,
+    A: mammos_entity.Entity | astropy.units.Quantity | numbers.Number | np.ndarray,
+    K1: mammos_entity.Entity | astropy.units.Quantity | numbers.Number | np.ndarray,
+    model: str = "cube50_singlegrain_random_forest_v0.1",
+) -> bool | np.ndarray:
+    """Classify material as soft or hard magnetic from micromagnetic parameters.
+
+    This function classifies a magnetic material as either soft or hard magnetic
+    based on its micromagnetic parameters spontaneous magnetization Ms, exchange
+    stiffness constant A and uniaxial anisotropy constant K1.
+    The shape of the input parameters needs to be the same. If single values are
+    provided, a single classification is returned. If arrays are provided, a
+    numpy array with the same shape is returned.
+
+    Different models are available for the prediction. The following list provides an
+    overview:
```

Comment:
Maybe just

```suggestion
    The following models are available for the prediction:
```

(It's a little less awkward with just the single model that we currently have)